### PR TITLE
Integrate Google Translate API to translate sentence and add CSS

### PR DESCRIPTION
**Pull request by emilyzhaoo**

### CODE REVIEW

File: portfolio/src/main/webapp/script.js [hunk 1]
```diff
@@ -33,6 +33,7 @@ function loadTasks() {
     // get user selection for quantity number
     var quantity = getSelect(); 
 
+
```
Comment (azhang72) [inline portfolio/src/main/webapp/script.js]: nit: remove extra empty line

File: portfolio/src/main/webapp/script.js [hunk 2]
```diff
@@ -99,4 +107,24 @@ function drawChart() {
...
+    params.append('text', text);
+    params.append('languageCode', languageCode);
+
+    fetch('/translate', { method: 'POST', body: params}).then(response => 
+        response.text()).then((translatedMessage) => {
+          // Sets header to the translated message
+          document.getElementById('text').innerText = translatedMessage;
+          loadingContainer.innerText = '';
+        });
 }
```
Comment (azhang72) [inline portfolio/src/main/webapp/script.js]: nit: newline at end of file.

File: portfolio/src/main/webapp/script.js [hunk 3]
```diff
@@ -99,4 +107,24 @@ function drawChart() {
...
         chart.draw(data, options);
     });
+}
+
+/** Requests the translation */
+function requestTranslation() {
+    const text = document.getElementById('text').textContent;
+    const languageCode = document.getElementById('language').value;
+
+    const loadingContainer = document.getElementById('result');
```
Comment (azhang72) [inline portfolio/src/main/webapp/script.js]: Why does the |loadingContainer| have id "result"?

**Review by azhang72 — CHANGES_REQUESTED** [review]

File: portfolio/src/main/java/com/google/sps/servlets/TranslateServlet.java [hunk 4]
```diff
@@ -0,0 +1,46 @@
+// Copyright 2019 Google LLC
```
Comment (EkkiD) [inline portfolio/src/main/java/com/google/sps/servlets/TranslateServlet.java]: nit:2020

File: portfolio/src/main/java/com/google/sps/servlets/TranslateServlet.java [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/translate")
+public class TranslateServlet extends HttpServlet {
+
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    // Get the request parameters.
+    String originalText = request.getParameter("text");
+    String languageCode = request.getParameter("languageCode");
```
Comment (EkkiD) [inline portfolio/src/main/java/com/google/sps/servlets/TranslateServlet.java]: consider renaming this a parameter to make it clear whether this is the source language or the destination language

File: portfolio/src/main/webapp/index.html [hunk 6]
```diff
@@ -4,7 +4,7 @@
     <meta charset="UTF-8">
     <title style="text-align:center">My Portfolio</title>
     <link rel="stylesheet" href="style.css">
-    <script src="script.js"></script>
+    <script src="script.js"></script>
```
Comment (EkkiD) [inline portfolio/src/main/webapp/index.html]: nit: remove trailing spaces

File: portfolio/src/main/webapp/script.js [hunk 7]
```diff
@@ -56,7 +56,7 @@ function createTaskElement(task) {
     const sentence = document.createElement('span');
     sentence.innerText = ("Last Sunday, I was " + task.verb + " and I saw this " 
     + task.adj + " " + task.animal + ", who was also " + task.verb +"."); 
-
+
```
Comment (EkkiD) [inline portfolio/src/main/webapp/script.js]: nit: remove trailing spaces

File: portfolio/src/main/java/com/google/sps/servlets/TranslateServlet.java (hunk 5 above)
Comment (EkkiD) [reply portfolio/src/main/java/com/google/sps/servlets/TranslateServlet.java]: or "original" and "target"... whatever terminology is best suited in your mind

**Review by EkkiD — CHANGES_REQUESTED** [review]

File: portfolio/src/main/java/com/google/sps/servlets/TranslateServlet.java [hunk 8]
```diff
@@ -0,0 +1,46 @@
...
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/translate")
+public class TranslateServlet extends HttpServlet {
+
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    // Get the request parameters.
+    String originalText = request.getParameter("text");
+    String targetLanguage = request.getParameter("languageCode");
```
Comment (EkkiD) [inline portfolio/src/main/java/com/google/sps/servlets/TranslateServlet.java]: I think it's worth renaming the parameter here too. It makes the API easier to use for people who might be less familiar with it.

**Review by EkkiD — CHANGES_REQUESTED** [review]

File: portfolio/src/main/webapp/index.html [hunk 9]
```diff
@@ -104,5 +119,4 @@ <h2 class="center" style="width:75%">Active COVID-19 Cases in the World</h2>
        <a href="www.linkedin.com/in/emily-zhaoo" class="fa fa-linkedin"></a>
     </div>
 </footer>
-</html>
- 
\ No newline at end of file
+</html>
```
Comment (azhang72) [inline portfolio/src/main/webapp/index.html:122]: nit: newline at end of file

**Review by azhang72 — APPROVED** [review]
lgtm with a nit

File: portfolio/src/main/webapp/script.js (hunk 3 above)
Comment (azhang72) [reply portfolio/src/main/webapp/script.js]: Did you miss this?

**Review by EkkiD — APPROVED** [review]
LGTM

**Pull request merged**